oap-server/server-fetcher-plugin/kafka-fetcher-plugin/src/main/java/org/apache/skywalking/oap/server/analyzer/agent/kafka/KafkaFetcherHandlerRegister.java

```diff
@@ -137,7 +137,11 @@ public void run() {
                 Iterator<ConsumerRecord<String, Bytes>> iterator = consumerRecords.iterator();
                 while (iterator.hasNext()) {
                     ConsumerRecord<String, Bytes> record = iterator.next();
-                    handlerMap.get(record.topic()).handle(record);
+                    try {
+                        handlerMap.get(record.topic()).handle(record);
+                    } catch(Throwable t) {
+                        log.error("consume record error", t);
```

Comment:
When and why this exception happens?

Reply:
aha ,the log is lost. In my situation , `ServiceManagementHandler.java`  `L65` throw a NPE.
```
    @Override
    public void handle(final ConsumerRecord<String, Bytes> record) {
        try {
            if (record.key().startsWith("register-")) {
                serviceReportProperties(InstanceProperties.parseFrom(record.value().get()));
            } else {
                keepAlive(InstancePingPkg.parseFrom(record.value().get()));
            }
        } catch (InvalidProtocolBufferException e) {
            log.error("", e);
        }
    }
```
Maybe a record without a key?

Reply:
I am fine if you just want to make sure the illegal data would not break all things. 
At the same time, fix doc and ci.

Reply:
As you show the code, I suggest changing the `InvalidProtocolBufferException` to `Throwable` or `Exception`. And then don't need to add try-catch statement again.

Reply:
There mayby Another `Handler`  throws a uncaputure exception.

Reply:
Yeap, I mean all handlers should be modified. :)

Reply:
i think every hander should catch `Exception` not `InvalidProtocolBufferException`.